Resources/en.lproj/Localizable.strings

```diff
@@ -1202,14 +1201,14 @@
 "toggle_public_transport" = "Show/hide public transport";
 "public_transport_show" = "Show public transport";
 "public_transport_hide" = "Hide public transport";
-"quick_action_showhide_mapillary_title" = "Show/hide Mapillary";
```

Comment:
Revert this change and paste the correct text into quick_action_showhide_mapillary_descr